omx/gstomxvideodec.c

```diff
@@ -201,53 +206,6 @@ gst_omx_video_dec_open (GstVideoDecoder * decoder)
 
   GST_DEBUG_OBJECT (self, "Opened decoder");
 
-#if defined (USE_OMX_TARGET_RPI) && defined (HAVE_GST_GL)
```

Comment:
I would leave all of these!

Reply:
@michaelgruner  I removed RPI and GL code because this repo (and branch) is for DM81xx, so the program never enters RPI/GL macros. I wonder if you prefer to leave it to keep everything more generic ?

Reply:
@michaelgruner Restored non-ti target code 1fdec4363f2c6a44b856e4cfc69c88ef4ef36503